socs/agents/acu/agent.py

```diff
@@ -306,6 +336,7 @@ def monitor(self, session, params):
         session.data = {'PlatformType': self.acu_config['platform'],
                         'DefaultScanParams': self.scan_params,
                         'StatusResponseRate': 0.,
+                        'IgnoredAxes': self.ignore_axes,
```

Comment:
Add to session.data example in docstring.

Reply:
done